packages/webapp/webapp_tests.js

```diff
@@ -156,23 +156,6 @@ Tinytest.add("webapp - generating boilerplate should not change runtime config",
   test.isFalse(__meteor_runtime_config__.WEBAPP_TEST_KEY);
 });
 
-// Support 'named pipes' (strings) as ports for support of Windows Server / Azure deployments
-Tinytest.add("webapp - port should be parsed as int unless it is a named pipe", function(test){
```

Comment:
@abernix, in my previous commit https://github.com/meteor/meteor/commit/693ac8ae7293fdbd0dbfaca47b67fe39a950994d, I preserved the legacy parsePort() behaviour and tests, but you weren't happy because socket path would require an absolute path starting with a forward slash, i.e. you could not create a socket file in the current directory.

I note that I have seen other frameworks that use the presence of a forward slash to distinguish between listening on a UNIX socket and listening on a TCP port.

In order to satisfy your concern, I changed this behaviour in commit https://github.com/meteor/meteor/commit/d9e7f8672b62217d69056c37ae6e00fe412cffb3 so that a number would be treated as a TCP port and anything else would be treated as a UNIX socket, and this required removing the existing parsePort() functionality.